ogc-rs/src/audio.rs

```diff
@@ -1,7 +1,3 @@
-//! The ``audio`` module of ``ogc-rs``.
-//!
-//! This module implements a safe wrapper around the audio functions found in ``audio.h``.
-
```

Comment:
Any reason to delete the simple documentation headers of the files? This seems to happen in other files as well.

Reply:
I think I did it earlier because it was redundant in `cargo doc`. I'll revert it.